src/functions/sendCandidateResults/application/service/categories/BE/fault-provider-cat-be.ts

```diff
@@ -0,0 +1,127 @@
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import { Fault } from '../../../../domain/fault';
+import { CompetencyOutcome } from '../../../../domain/competency-outcome';
+import {
+  convertBooleanFaultObjectToArray,
+  convertNumericFaultObjectToArray,
+  getCompletedManoeuvres,
+} from '../../fault-provider';
+import { Competencies } from '../../../../domain/competencies';
+import { QuestionOutcome, QuestionResult } from '@dvsa/mes-test-schema/categories/common';
+
+export const getDrivingFaultsCatBE = (testData: CatBEUniqueTypes.TestData | undefined): Fault [] => {
```

Comment:
again as it's a category specific provider we can remove the CatBE from the method names.

Reply:
The reason i've appended the category to the function is this is used in a switch statement, so it gave clarity to what function was being called